spec/models/user_spec.rb

```diff
@@ -245,4 +245,116 @@
       end
     end
   end
+
+  describe ".find_by_get_an_identity_id" do
+    let(:uid) { "foobar" }
+    let!(:user) { create(:user, :with_get_an_identity_id, uid:) }
+
+    it "returns the user with matching uid from the with_get_an_identity_id scope" do
+      expect(User.find_by_get_an_identity_id(uid)).to eq(user)
+    end
+
+    it "returns nil if no user matches the uid" do
+      expect(User.find_by_get_an_identity_id("nonexistent-uid")).to be_nil
```

Comment:
```ruby
User.find_by_get_an_identity_id(nonexistant_uid)
```

Reply:
If the variable is not used anywhere, I personally prefer to use it inline. However, happy to move it to let - but then is there a some sort of guidance, or maybe we could configure rubocop to catch this kind of things?